redis/src/test/scala/zio/redis/KeysSpec.scala

```diff
@@ -1,20 +1,31 @@
 package zio.redis
 
+import java.time.{ Duration => JavaDuration }
+
 import zio.clock.Clock
 import zio.duration._
+import zio.logging.Logging
+import zio.redis.RedisError.ProtocolError
 import zio.test.Assertion._
 import zio.test.TestAspect._
 import zio.test._
 import zio.test.environment.{ TestClock, TestConsole, TestRandom, TestSystem }
-import zio.{ Chunk, Has, ZIO }
+import zio.{ Chunk, Has, ZIO, ZLayer }
 
 trait KeysSpec extends BaseSpec {
 
   val keysSuite: Spec[Has[Clock.Service] with Has[RedisExecutor.Service] with Has[TestClock.Service] with Has[
     TestConsole.Service
   ] with Has[TestRandom.Service] with Has[TestSystem.Service] with Has[RedisExecutor.Service] with Has[
     Annotations.Service
-  ], TestFailure[RedisError], TestSuccess] =
+  ], TestFailure[RedisError], TestSuccess] = {
+
+    // Note the migrate command tests require two Redis instances, so here we create a second
+    // Redis RedisExecutor
+    val secondConfigLayer  = ZLayer.succeed(RedisConfig("localhost", 6380))
+    val secondRedisService = (Logging.ignore ++ secondConfigLayer >>> RedisExecutor.live).fresh
+    val migrateTimeout     = JavaDuration.ofMillis(5000)
```

Comment:
I think you could use zio's duration here. Also, the constants can go to companion.